src/sql/parts/query/editor/charting/chartView.ts

```diff
@@ -20,7 +20,7 @@ import { Taskbar } from 'sql/base/browser/ui/taskbar/taskbar';
 import { ChartType } from 'sql/workbench/parts/dashboard/widgets/insights/views/charts/interfaces';
 import { Registry } from 'vs/platform/registry/common/platform';
 import * as DOM from 'vs/base/browser/dom';
-import { SelectBox } from 'vs/base/browser/ui/selectBox/selectBox';
+import { SelectBox } from 'sql/base/browser/ui/selectBox/selectBox';
```

Comment:
Changing the entire class seems risky. Have to tested all the boxes and make sure they work properly?

Reply:
yes, I tested all the select boxes.